submitter/biosamples_submitter_service.py

```diff
@@ -0,0 +1,27 @@
+from submission_broker.services.biosamples import BioSamples
+from submission_broker.submission.entity import Entity
+
+ENV = 'dev'
+BIOSTUDIES_STUDY_LINK = f"https://www{ENV}.ebi.ac.uk/biostudies/studies/"
```

Comment:
How will this work when we are in prod?

Reply:
Ah, sorry, I left it there. I have to fix that. Good catch, tx!